### ISSUE TITLE
Attempting to craft floating candles CTDs

### ISSUE THREAD

**Issue by Java-Queen** [DrasticDemise/Hubris#15]
With black candles in the Arcane Worktable (I've not tried other candle colors), placing the air shards in the table to produce Floating Candles (from thaumic esoterica) causes a crash. Repeated multiple times, reopening the table with the materials still in it triggers the crash again.

http://pastebin.com/0kuatJ1a

**Comment by DrasticDemise** [comment]
Interesting. I will look into it, thanks!

**Comment by Patzos** [comment]
I think there is a general problem with arcane worktable and modded recipies. I was using them instead of the vanilla ones cause they hold inventory when closed. So many times I have crashed when SHIFT CLICK extracting output products. After restarting i found the recipe slightly differently arranged in the grid.
TL;DR - Are you shift-taking the candles out? Try not to.
